test/dynamo/test_functions.py

```diff
@@ -238,6 +238,26 @@ def test_listarg5(a, b):
         kwargs = {"dim": 0}
         return torch.cat(*args, **kwargs)
 
+    @make_test
+    def test_deque(a, b):
+        d = collections.deque([a, b])
```

Comment:
How does this work if collections is in skipfiles?

Reply:
Answer: Skipfiles variable, although its kinda sus and not in the spirit of skipfiles